README.md

```diff
@@ -1,21 +1,43 @@
-# The IPFS User Interface Project
+# The IPFS GUI Team
 
-> Make the IPFS GUIs simple, accessible, reusable and beautiful. That is the IPFS GUI Team! 
+> The meeting point for making great Graphical User Interfaces.
 
 ## Goal
 
 Make the IPFS GUIs simple, accessible, reusable and beautiful.
 
+- **Simple** - Fight complexity at every step. The are big ideas in IPFS. Let's present them clearly.
+- **Accessible** - IPFS **must** be useable and comprehensible for everyone.
```

Comment:
`usable`

Reply:
oh my, i've been spelling that wrong for some time.